tests/scripts/thread-cert/border_router/MATN_05_ReregistrationToSameMulticastGroup.py

```diff
@@ -0,0 +1,244 @@
+
+        # 6a. By internal means, Router_1 stops listening to the multicast
+        # address, MA1.
+        router1.del_ipmaddr(MA1)
+
+        # 7. After (MLR_TIMEOUT_MIN+2) seconds, Host multicasts a ping packet to
+        # multicast address, MA1, on the backbone link.
+        self.simulator.go(consts.MLR_TIMEOUT_MIN + 4)
+        self.assertFalse(
+            host.ping(MA1, backbone=True, ttl=10, interface=host.get_ip6_address(config.ADDRESS_TYPE.ONLINK_ULA)[0]))
+
+        self.collect_ipaddrs()
+        self.collect_rloc16s()
+        self.collect_rlocs()
+        self.collect_leader_aloc(BR_1)
+        self.collect_extra_vars()
+
+    def verify(self, pv: pktverify.packet_verifier.PacketVerifier):
+        pkts = pv.pkts
+        vars = pv.vars
+        pv.summary.show()
+        logging.info(f'vars = {vars}')
+
+        # Ensure the topology is formed correctly
+        pv.verify_attached('Router_1', 'BR_1')
+        pv.verify_attached('BR_2')
+
+        # Initial registration
+        # Router_1 registers for multicast address, MA1, at BR_1.
+        # Router_1 unicasts an MLR.req CoAP request to BR_1 as
+        # "coap://[<BR_1 RLOC or PBBR ALOC>]:MM/n/mr".
+        # The payload contains "IPv6Address TLV: MA1".
+        initial_registration_pkt = pkts.filter_wpan_src64(vars['Router_1']) \
+            .filter_ipv6_2dsts(vars['BR_1_RLOC'], vars['LEADER_ALOC']) \
+            .filter_coap_request('/n/mr') \
+            .filter(lambda p: p.thread_meshcop.tlv.ipv6_addr == [MA1]) \
+            .must_next()
+
+        # 1. Host sends a ping packet to the multicast address, MA1.
+        _pkt = pkts.filter_eth_src(vars['Host_ETH']) \
+            .filter_ipv6_dst(MA1) \
+            .filter_ping_request() \
+            .must_next()
+
+        # 2. BR_1 forwards the ping packet with multicast address, MA1, to its
+        # Thread Network encapsulated in an MPL packet.
+        pkts.filter_wpan_src64(vars['BR_1']) \
+            .filter_AMPLFMA(mpl_seed_id=vars['BR_1_RLOC']) \
+            .filter_ping_request(identifier=_pkt.icmpv6.echo.identifier) \
+            .must_next()
+
+        # 3. Router_1 receives the MPL packet containing an encapsulated ping
+        # packet to MA1, sent by Host, and unicasts a ping response packet back
+        # to Host.
+        pkts.filter_wpan_src64(vars['Router_1']) \
+            .filter_ipv6_dst(_pkt.ipv6.src) \
+            .filter_ping_reply(identifier=_pkt.icmpv6.echo.identifier) \
+            .must_next()
+
+        # 3a. Within MLR_TIMEOUT_MIN seconds of initial registration, Router_1
+        # re-registers for multicast address, MA1, at BR_1.
+        # Router_1 unicasts an MLR.req CoAP request to BR_1 as
+        # "coap://[<BR_1 RLOC or PBBR ALOC>]:MM/n/mr".
+        # The payload contains "IPv6Address TLV: MA1".
+        pkts.copy() \
+            .filter_wpan_src64(vars['Router_1']) \
+            .filter_ipv6_2dsts(vars['BR_1_RLOC'], vars['LEADER_ALOC']) \
```

Comment:
```suggestion
            .filter_ipv6_2dsts(vars['BR_1_RLOC'], vars['LEADER_ALOC']) \
```
Need to use PBBR ALOC instead of LEADER ALOC. 
Same issue in `MATN-TC-02` and `MATN-TC-03`.